MiniApp/Classes/core/Protocols/MiniAppUserInfoDelegate.swift

```diff
@@ -112,3 +119,15 @@ extension Date {
         return Int(timeSince1970 * 1000)
     }
 }
+
+public class MAPoints: Codable {
+    let pointsStandard: Int
+    let pointsTerm: Int
+    let pointsCash: Int
+
+    public init(pointsStandard: Int, pointsTerm: Int, pointsCash: Int) {
+        self.pointsStandard = pointsStandard
+        self.pointsTerm = pointsTerm
+        self.pointsCash = pointsCash
+    }
+}
```

Comment:
I think we should start to regroup all these small classes in a separate file some time.

Reply:
yeah true, should I do it right away or should we think of that anytime later?

Reply:
That should be fine, it can be part of a refactor PR ;)